contrib/NeonVM/string.c

```diff
@@ -1,6 +1,6 @@
 #ifdef _MSC_VER
 #pragma warning (push)
-#pragma warning (disable: 4001)
+#pragma warning (disable: 4001)     /* Disable sing-line comment warning */
```

Comment:
You're only using single line comments in one place. Do you really need to disable this warning?

Reply:
Will remove all of these, project-wide in a future push.